stl/inc/sstream

```diff
@@ -371,16 +371,16 @@ protected:
 
             // fallthrough
         default:
-            return pos_type(off_type(-1));
+            return static_cast<pos_type>(off_type(-1));

```

Comment:
I saw some occurences where we simply return -1;

I see that `pos_type` and `off_type` are marked as implementation defined. Can we assume that they are under our control and simply return -1 because both are `long long`?

Reply:
Maybe use `pos_type{off_type{-1}}` and follow the recomendation?

Reply:
these come from traits, so may be user defined type. reverted this rule